tests/bootstrap.php

```diff
@@ -24,8 +24,6 @@
 
 require dirname(__DIR__) . '/vendor/autoload.php';
 
-if (file_exists(dirname(__DIR__) . '/config/bootstrap.php')) {
-    require dirname(__DIR__) . '/config/bootstrap.php';
-} elseif (method_exists(Dotenv::class, 'bootEnv')) {
```

Comment:
just to check, and what if method doesn't exist -> just fail? no need to require boostrap file?

Reply:
That file doesn't not exist anymore in Symfony 5. About the method check, I removed the check, you are right, it cannot work without it.